### ISSUE TITLE
Open_Browser keyword raises exception and webdriver not being disposed

### ISSUE THREAD

**Issue by blinkins0n** [robotframework/SeleniumLibrary#304]
Running robotframework-selenium2library on Windows 2008, I was getting timeout in the browser.get(url) below when invoking chromedriver.exe. Suggest wrapping the call with try catch browser.quit() invocation.

_browsermanagement.py

```
def open_browser(self, url, browser='firefox', alias=None,remote_url=False,
            desired_capabilities=None,ff_profile_dir=None):
    """
    """
    ..................
    try:
        browser.get(url)
        self._debug('Opened browser with session id %s'
                   % browser.session_id)
        return self._cache.register(browser, alias)
    except:
        print 'BrowserManagement: exception. Could not load the page. Browser and    service will be stopped...'
        browser.quit()        
        raise RuntimeError("BrowserManagement: exception. Could not load the page.")
```

**Comment by lawred2** [comment]
Excellent... I was having an issue where the chromedriver process was not being unloaded and remained orphaned at the end of a test. I think this could have possibly been the root cause. I'll make this change locally and will see if it makes a difference.

**Comment by blinkins0n** [comment]
Hi lawred2,

I just realized that bug fix may have been released in the merge below:
https://github.com/rtomac/robotframework-selenium2library/commit/9e162827be0e7dfe09b638d3893f3ef66411bb2c

Regards.

**Comment by ombre42** [comment]
Thank you for reporting this. Closing as duplicate of #70, which was fixed in PR #263 for 1.6.
